spec/controllers/admin/categories_controller_spec.rb

```diff
@@ -11,6 +11,11 @@
     request.session = { :user => henri.id }
   end
 
+  it "test_new" do
```

Comment:
I believe that editing is covered in the pre-existing `describe "test_edit"` block lower down.